cockatrice/src/tablezone.cpp

```diff
@@ -55,7 +55,15 @@ bool TableZone::isInverted() const
 
 void TableZone::paint(QPainter *painter, const QStyleOptionGraphicsItem * /*option*/, QWidget * /*widget*/)
 {
-    painter->fillRect(boundingRect(), themeManager->getTableBgBrush());
+    QBrush brush;
+
+    brush = themeManager->getTableBgBrush();
+    // If the player is other than Player 1
+    if (player->getId() > 0) {
+        // The palyer's id starts with 0 so in order to get the correct image we need to add 1
+        brush = themeManager->getExtraTableBgBrush(QString::number(player->getId() + 1));
```

Comment:
What happens if this misses? (read: What if the images aren't found?)

Reply:
I've tested it and it just uses the default images